packages/snaps-controllers/package.json

```diff
@@ -45,7 +45,7 @@
     "@metamask/approval-controller": "^4.0.0",
     "@metamask/base-controller": "^3.2.0",
     "@metamask/json-rpc-engine": "^7.1.1",
-    "@metamask/object-multiplex": "^1.2.0",
+    "@metamask/object-multiplex": "^2.0.0",
```

Comment:
@legobeat Can we bump this across all the snaps monorepo packages?

Reply:
Our Yarn constraints require that we use the same dependency versions across all packages.

Reply:
Hmm could be that providers v14 needs some more attention to bump 🤔